mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -71,6 +71,22 @@ private[recommendation] trait ALSModelParams extends Params with HasPredictionCo
 
   /** @group getParam */
   def getItemCol: String = $(itemCol)
+
+  /**
+   * Param for strategy for dealing with unknown users or items at prediction time.
+   * Supported values:
```

Comment:
Should we add a note here about this being useful/necessary when performing cross validation or evaluating the model on unseen data?